### PR TITLE
Added Raspberry Pi models

### PR DESCRIPTION
**Pull request by kenvandine**
Adds new models for the pi

### CODE REVIEW

**Comment by kenvandine** [comment]
I've submitted the models to be signed

**Comment by kenvandine** [comment]
The signed models are now included, but we need the transfer request for the pi-desktop snap to be processed before images can be built from the signed model.  For reference https://forum.snapcraft.io/t/transfer-request-pi-desktop-to-canonical/37625

**Comment by kenvandine** [comment]
pi-desktop has been transfer to Canonical, this should be good now.

**Review by robert-ancell — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -15,6 +15,17 @@ pc-dangerous.img: ubuntu-core-desktop-22-amd64-dangerous.model $(EXTRA_SNAPS)
 	  $(foreach snap,$(ALL_SNAPS),--snap $(snap)) $<
 	mv dangerous/pc.img pc-dangerous.img
 
+pi.img: ubuntu-core-desktop-22-pi.model $(EXTRA_SNAPS)
+	rm -rf dangerous/
+	ubuntu-image snap --output-dir img --image-size 12G \
```
Comment (robert-ancell) [inline Makefile:20]: Note the PC images are now 20G, it 12G still appropriate for Pi?

Comment (kenvandine) [reply Makefile:20]: Yes, I think so.  For the Pi we can assume they will be written to an SD card directly.  The pc.img could be booted in a VM directly, and at 12G had only a couple hundred megs free.

**Pull request merged**